pyproject.toml

```
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "jspsych"
+version = "7.2.2"
+description = "A JavaScript library for creating behavioral experiments in a web browser"
+authors = ["Josh de Leeuw <josh.deleeuw@gmail.com>"]
+license = "MIT"
```

The version here is not automatically updated. Since this section is intended for publishing on PyPI and we're not doing that, we might as well make that clear by mocking it like
```suggestion
version = "0"
description = ""
authors = []
```
WDYT?